pkgs/development/python-modules/secretstorage/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, pkgs, buildPythonPackage, dbus-python, python, cryptography }:
+
+buildPythonPackage rec {
+  pname = "secretstorage";
+  version = "2.3.1";
+  name = "${pname}-${version}";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "mitya57";
+    repo = "secretstorage";
+    rev = "2636a47b45aff51a21dfaf1cbd9f1f3c1347f7f4";
+    sha256 = "1xs35ywqxxriwhm79kvivad0hszcm4ah29v34vbshlqvc66qr171";
+  };
+
+  postPatch = "patchShebangs .";
+  propagatedBuildInputs = [ dbus-python cryptography ];
+  doCheck = false;
+
+  meta = with stdenv.lib; {
```

Comment:
Did you want to add yourself as maintainer?

Reply:
not really. I am not proficient with nixos nor I am knowledgeable about dbus/keyring stuff. If it's mandatory, I can put myself as a best effort thing.

Reply:
I think it is mandatory, but maybe someone else will want to be a maintainer. However, it shouldn't be too hard to be a maintainer, it should mainly just be about updating the revision and sha256 whenever there's a new release, and checking to see if it builds/works.